layers/+lang/scheme/packages.el

```diff
@@ -102,17 +102,44 @@
         "sR" 'geiser-eval-region-and-go
         "ss" 'geiser-set-scheme)
 
+      (evil-define-key 'insert 'geiser-repl-mode-map
+        (kbd "S-<return>") 'geiser-repl--newline-and-indent)
+
       (evil-define-key 'normal 'geiser-repl-mode-map
-      "gj" 'geiser-repl-next-prompt
-      "gk" 'geiser-repl-previous-prompt)
+        "]]" 'geiser-repl-next-prompt
+        "[[" 'geiser-repl-previous-prompt
+        "gj" 'geiser-repl-next-prompt
+        "gk" 'geiser-repl-previous-prompt)
 
+      (spacemacs/declare-prefix-for-mode 'geiser-repl-mode "mh" "help")
       (spacemacs/set-leader-keys-for-major-mode 'geiser-repl-mode
-        "q" 'geiser-repl-exit
+        "b" 'geiser-pop-symbol-stack
         "c" 'geiser-repl-clear-buffer
         "i" 'geiser-repl-interrupt
+        "f" 'geiser-load-file
+        "l" 'geiser-insert-lambda
         "m" 'geiser-repl-import-module
         "u" 'geiser-repl-unload-function
-        "h" 'geiser-doc-symbol-at-point))))
+        "hd" 'geiser-doc-symbol-at-point
+        "s" 'geiser-squarify
+        "q" 'geiser-repl-exit
+        "hc" 'geiser-customize-implementation)
+
+      (evilified-state-evilify-map geiser-doc-mode-map
```

Comment:
do you think we should also added bindings for forward-button and backward-button, as it does in evil-collections? (I've no strong preference on this one)

Reply:
Well generally I use `o` for this, and otherwise personally I prefer `TAB/S-TAB` which are already the default emacs (i.e. evilified) bindings. But indeed I should at least add `TAB/S-TAB` to the docs. Or do you think I should add evil-collection's `C-j/C-k` also?

Reply:
C-j/C-k is almost as common as TAB/S-TAB in various evil bindings.
So you can add both.